### PR TITLE
Quick player-count balance for QRF/singleAttack/wavedCA

### PR DESCRIPTION
**Pull request by jaj22**
### What type of PR is this.
1. [ ] Bug
2. [X] Change
3. [ ] Enhancement

### What have you changed and why?
Softer version of #2079 based on the principle that aggro is already somewhat player-count dependent, so it should be used additively rather than multiplicatively with direct player count balancing. Uses a bit of randomness to smooth out the boundaries. Typical vehicle counts:

QRF:
1 player, 25 aggro: 1.5 vehicles.
9 players, 50 aggro: 2.5 vehicles.
26 players, 100 aggro: 3.5 vehicles.

singleAttack (typically higher aggro because a marker was just captured):
1 player, 50 aggro: 3 vehicles.
9 players, 75 aggro: 4.8 vehicles.
26 players, 100 aggro: 7.5 vehicles.

wavedCA (+2 vehicles for first wave)
1 player: 5.5 vehicles first wave, 3.5 subsequent
9 players: 7 vehicles first wave, 5 subsequent
26 players: 10 vehicles first wave, 8 subsequent

wavedCA isn't aggro-dependent because it's kinda odd to increase the attack size because you just took a marker elsewhere. It's a bit flatter than the others because the garrisons + statics are more useful.

### Please specify which Issue this PR Resolves.
Supersedes the other half of #2079

### Please verify the following and ensure all checks are completed.
1. [X] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [X] No
2. [ ] Yes (Please provide further detail below.)

### CODE REVIEW

**Comment by jaj22** [comment]
Improved the logging a bit, and adjusted occ vs inv behaviour so that it has no dependence on player count. This wasn't easily possible for QRFs because the function doesn't actually know the target side. Not very important, will be ironed out eventually.

File: A3-Antistasi/functions/CREATE/fn_wavedCA.sqf [hunk 1]
```diff
@@ -163,12 +159,12 @@ private _uav = objNull;
 while {(_waves > 0)} do
 {
 	_soldiers = [];
-	_nVeh = 2 + random (2) + (_aggro / 25);
-	_nVeh = _nVeh + (skillMult - 2);
+	private _playerScale = (8 + count (allPlayers - entities "HeadlessClient_F")) / 17;
```
Comment (HakonRydland) [inline A3-Antistasi/functions/CREATE/fn_wavedCA.sqf:162]: should we maybe make this a function?

Comment (jaj22) [reply A3-Antistasi/functions/CREATE/fn_wavedCA.sqf:162]: Global variable, if anything (it's fine to update balance values on a 10-minute cycle), but I'm not intending it to be around for very long. The version in the (proper) balance rework is handled differently due to the interaction with aggression, so all this gets gradually replaced.

**Review by HakonRydland — APPROVED** [review]

File: A3-Antistasi/functions/CREATE/fn_wavedCA.sqf (hunk 1 above)
Comment (jaj22) [reply A3-Antistasi/functions/CREATE/fn_wavedCA.sqf]: Ah whatever, doesn't hurt to pull it out into a function.

**Pull request merged**